Export both supported LZO public-header include styles in pkg-config.

TODO: propose the build-system interface/fix upstream.

--- a/lzo2.pc.cmakein
+++ b/lzo2.pc.cmakein
@@ -17,5 +17,5 @@
 Version: @PROJECT_VERSION@
 Description: LZO - a real-time data compression library
 URL: http://www.oberhumer.com/opensource/lzo/
-Cflags: -I${includedir}/lzo
+Cflags: -I${includedir} -I${includedir}/lzo
 Libs: -L${libdir} -llzo2
